scripts/ps-modules/Util/Util.psm1

```diff
@@ -54,6 +54,16 @@ function Get-IsOnLinux {
     $IsLinux
 }
 
+function Get-OSPackageSection {
+    if ((Get-IsOnWindowsOS)) {
+        return "win"
+    }
+    if ((Get-IsOnMacOS)) {
+        return "mac"
+    }
+    return "linux"
+}
+
```

Comment:
Nitpick: This name might be a bit misleading as to what it does, since it it just getting what OS it is on, and has nothing to do with a package section in the function.  We happen to be using it to get the package section in the calling function, but that is not what it this does.

A future package section I'd like to see us add would be one to transpile to WASM.  In that case, the package section in the json would not be tied to the OS, as you could perform this on various OSes.

Reply:
I've updated to your original suggestion,`Get-OSType`.  Naming is hard and I agree that this was a bit misleading.